simple_classifier/Classifiers/Display.py

```diff
@@ -0,0 +1,15 @@
+from simple_classifier.Classifiers.ClassifierProfiler import ClassifierProfiler
+
+
+class Display:
+    def dump(self):
```

Comment:
`dump()`, `json()`, and `plot()` should take some `np.array`-like data as input, we could leave them aside for now till we decide the output of `ClassifierProfiler.profile_classifier()`.